homeassistant/components/telegram_bot/__init__.py

```diff
@@ -175,6 +177,8 @@ def load_data(hass, url=None, filepath=None, username=None, password=None,
                     params["auth"] = HTTPDigestAuth(username, password)
                 else:
                     params["auth"] = HTTPBasicAuth(username, password)
+            if verify_ssl is not None:
```

Comment:
This should be enough:
```suggestion
            if verify_ssl:
```

Reply:
Sure, thanks! I used the `is not None` construction because I saw it on other component and I thought it was part of your coding guidelines :). I'm totally new to HA development.

Reply:
I've just learned that you version was best practice, sorry for the bad advice.

Reply:
No worries :)

Reply:
@nbarrientos i checked this case and without `is not None` configuration `verify_ssl: false` doesn't work. Revert needed